lib/cylc/gui/view_tree.py

```diff
@@ -227,23 +229,43 @@ def sort_by_column(self, col_name=None, col_no=None, ascending=True):
             cols[col_no].set_sort_order(order)
             self.tmodelsort.set_sort_column_id(col_no - 1, order)
 
+    def _nat_convert(self, text):
+        return int(text) if text.isdigit() else text.lower()
+
+    def _nat_alpha_num_key(self, key):
+        return [self._nat_convert(c) for c in re.split('([0-9]+)', key)]
```

Comment:
Can compile the regex and store it as a constant to avoid having to recompile with each function call.

Reply:
Good spotting! Updating it...